src/features/metadata-based-view/MetadataBasedItemList.js

```diff
@@ -112,10 +122,11 @@ class MetadataBasedItemList extends React.Component<Props, State> {
 
     cellRenderer = ({ columnIndex, rowIndex, key, style }: CellRendererArgs): Element<'div'> => {
         const { hoveredRowIndex } = this.state;
+        const isFileNameCell = rowIndex > 0 && columnIndex === 1;
```

Comment:
Since these column indices appear more than once would it be helpful to create constants?